cmd/bootstrap/run/snapshot.go

```diff
@@ -0,0 +1,14 @@
+package run
+
+import (
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/state/protocol/inmem"
+)
+
+// GenerateRootProtocolSnapshot generates the root protocol snapshot to be used
```

Comment:
Do we need this?

Reply:
Ahh no, we don't need this. I started this before noticing it was already implemented in the `inmem` package. I will remove it.